### PR TITLE
ie11 fix + README.md change

### PR DESCRIPTION
**Pull request by phiden**
* fix for https://github.com/department-of-veterans-affairs/vets.gov-team/issues/3356
* Webpack changed some things, so you need a --public flag + to access the site on local network. Documentation: https://github.com/webpack/webpack-dev-server/releases/tag/v2.4.3

### CODE REVIEW

**Comment by ayaleloehr** [comment]
This never got merged and we just got a uservoice comment about this as well. Can we get the conflicts resolved here then this reviewed?

**Comment by phiden** [comment]
@ayaleloehr merge conflicts fixed -- looks like yarn.lock should probably be removed from the merge but I'm not sure how to do that; can you give me a hand, please?

**Comment by ayaleloehr** [comment]
Yarn.lock removed.

File: src/sass/modules/_m-homepage-hero.scss [hunk 1]
```diff
@@ -29,7 +29,8 @@
   font-weight: bold;
   font-size: 2.5rem;
   line-height: 1.3;
-  margin: auto;
+  //margin: auto;
```
Comment (cvalarida) [inline src/sass/modules/_m-homepage-hero.scss]: Could remove this.

**Review by cvalarida — APPROVED** [review]
Just tested in Chrome, IE 11, and Edge in Sauce Labs. Looks great!

**Pull request merged**